pubnub-kotlin/pubnub-kotlin-test/build.gradle.kts

```diff
@@ -44,11 +43,11 @@ kotlin {
 //            }
 //        }
 //
-//        val jvmMain by getting {
-//            dependencies {
-//                implementation(project(":pubnub-core:pubnub-core-impl"))
-//            }
-//        }
+        val jvmMain by getting {
+            dependencies {
+                api(kotlin("test-junit"))
```

Comment:
I guess that test-junit is base on old JUnit4.
What about using JUnit5
   testImplementation("org.junit.jupiter:junit-jupiter-api:5.8.1")
    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.8.1")

Reply:
I need the KMP dependency compatible with `kotlin-test` here, I can't use these other dependencies, and we're not accessing JUnit directly in our code, only through `kotlin-test`